src/k8s/cmd/util/hooks.go

```diff
@@ -0,0 +1,102 @@
+package cmdutil
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+const initialProcesEnvironmentVariables = "/proc/1/environ"
+
+// paths to validate if root in the owner
+var pathsOwnershipCheck = []string{"/sys", "/proc", "/dev/kmsg"}
+
+// HookVerifyResources checks ownership of dirs required for k8s to run.
+// HookVerifyResources if verbose true prints out list of potenital issues.
+// If potential issue found pops up warning for user.
+func HookVerifyResources(verbose bool) func(*cobra.Command, []string) {
+	return func(cmd *cobra.Command, args []string) {
```

Comment:
We should first check if we are running in a LXD container then check resources to determine if we are possibly missing the LXD profile.